### DISCUSSION TITLE
Escape sequence to get argument of last command.

### DISCUSSION THREAD

**Discussion by bruteForceAttack** [kovidgoyal/kitty#6849] [Q&A]
Hi, can somebody please help with the correct esc sequence so I can get the very last argument of previous command?
E.g.:

`ls foo`
`rm -rf "CMD+."`

Now shortcut _cmd+._ must give 'foo'.
Should be something like:

`map cmd+.  send_text all  XXXXX`.

However I am not entirely sure how to tell Kitty to convert "esc+." It's basically something like this: [alt-dot](https://superuser.com/questions/703911/iterm2-alt-dot-functionality-with-cmd-dot)

**Comment by kovidgoyal** [comment]
This is a shell feature not a terminal feature. The typical shortcut is
alt+. for this. If you are on macOS you need to set the
macos_option_as_alt setting in kitty.conf in order to have an alt key.

**Comment by bruteForceAttack** [comment]
Indeed, thanks for a switch answer. Works.

**Comment by bruteForceAttack** [comment]
Btw, is there any way how can you possibly do the same thing with cmd+. ? Something like mapping cmd+. to do the alt-dot action so it will do the same thing but only with some sort of redirection ?
